Any `func` command does not work

I installed the package globally, created a new project through VSCode extension and tried to run the sample without any changes. The error I get is:

```node
> func start

events.js:298
      throw er; // Unhandled 'error' event
      ^

Error: spawn C:\Users\apricot\AppData\Roaming\nvm\v13.7.0\node_modules\azure-functions-core-tools\bin/func ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:267:19)
    at onErrorNT (internal/child_process.js:467:16)
    at processTicksAndRejections (internal/process/task_queues.js:84:21)
Emitted 'error' event on ChildProcess instance at:
    at Process.ChildProcess._handle.onexit (internal/child_process.js:273:12)
    at onErrorNT (internal/child_process.js:467:16)
    at processTicksAndRejections (internal/process/task_queues.js:84:21) {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'spawn C:\\Users\\apricot\\AppData\\Roaming\\nvm\\v13.7.0\\node_modules\\azure-functions-core-tools\\bin/func',
  path: 'C:\\Users\\apricot\\AppData\\Roaming\\nvm\\v13.7.0\\node_modules\\azure-functions-core-tools\\bin/func',
  spawnargs: [ 'start' ]
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! hogehoge@1.0.0 start: `func start`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the hogehoge@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.        

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\apricot\AppData\Roaming\npm-cache\_logs\2020-02-05T09_13_13_764Z-debug.log
```

So, how about `func` itself?

```node
C:\Users\apricot\src\user\directory\hogehoge>func init fugafuga
events.js:298
      throw er; // Unhandled 'error' event
      ^

Error: spawn C:\Users\apricot\AppData\Roaming\nvm\v13.7.0\node_modules\azure-functions-core-tools\bin/func ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:267:19)
    at onErrorNT (internal/child_process.js:467:16)
    at processTicksAndRejections (internal/process/task_queues.js:84:21)
Emitted 'error' event on ChildProcess instance at:
    at Process.ChildProcess._handle.onexit (internal/child_process.js:273:12)
    at onErrorNT (internal/child_process.js:467:16)
    at processTicksAndRejections (internal/process/task_queues.js:84:21) {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'spawn C:\\Users\\apricot\\AppData\\Roaming\\nvm\\v13.7.0\\node_modules\\azure-functions-core-tools\\bin/func',
  path: 'C:\\Users\\apricot\\AppData\\Roaming\\nvm\\v13.7.0\\node_modules\\azure-functions-core-tools\\bin/func',
  spawnargs: [ 'init', 'fugafuga' ]
}
```

What happens in line 298...?
I installed both `2.7.2100` and `3.0.2106` but neither has worked.

@ankitkumarr Sorry for not responding to your mention for a while.

I have tried the steps you described in the latest comment but didn't succeed.  
(First of all you misattribute `npm .\lib\install.js` with `node .\lib\install.js` right?)

```node
apricot@apricot-R MINGW64 ~/AppData/Roaming/nvm/v13.7.0/node_modules/azure-functions-core-tools
$ node .\lib\install.js
internal/modules/cjs/loader.js:983
  throw err;
  ^

Error: Cannot find module 'C:\Users\apricot\AppData\Roaming\nvm\v13.7.0\node_modules\azure-functions-core-tools\.libinstall.js'
[90m    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:980:15)[39m
[90m    at Function.Module._load (internal/modules/cjs/loader.js:862:27)[39m
[90m    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)[39m
[90m    at internal/main/run_main_module.js:17:47[39m {
  code: [32m'MODULE_NOT_FOUND'[39m,
  requireStack: []
}
```

--- a/src/Azure.Functions.Cli/npm/package.json
+++ b/src/Azure.Functions.Cli/npm/package.json
@@ -30,10 +30,10 @@
         "chalk": "3.0.0",
         "command-exists": "1.2.8",
         "glob": "7.1.6",
-        "https-proxy-agent": "3.0.1",
+        "https-proxy-agent": "5.0.0",
         "progress": "2.0.3",
-        "rimraf": "3.0.0",
+        "rimraf": "3.0.2",
         "tmp": "0.1.0",
-        "unzipper": "0.10.8"
+        "unzipper": "0.10.10"
     }
 }

